enos/modules/test_cluster_health/scripts/allocs.sh

```diff
@@ -41,13 +41,17 @@ while true; do
         error_exit "Some allocs are not running:\n$(nomad alloc status -json | jq -r '.[] | select(.ClientStatus != "running") | .ID')"
     fi
 
-    echo "Running allocs: $running_allocs, expected $ALLOC_COUNT. Waiting for $elapsed_time  Retrying in $POLL_INTERVAL seconds..."
+    echo "Running allocs: $running_allocs, expected "$ALLOC_COUNT". Waiting for $elapsed_time  Retrying in $POLL_INTERVAL seconds..."
     sleep $POLL_INTERVAL
     elapsed_time=$((elapsed_time + POLL_INTERVAL))
 done
 
 echo "All ALLOCS are running."
 
+if [ "$allocs_length" -eq 0 ]; then
+    exit 0
+fi
```

Comment:
Aren't we already making this assertion if `$ALLOC_COUNT` isn't 0?

Reply:
if allocs are supposed to be 0, it doesn't fail and then it tries to stop an alloc from the 0 that are running and we end up with a div by 0

Reply:
Oh right, because for system jobs `ALLOC_COUNT` is 0!